static-assets/components/cstudio-dialogs/new-content-type.js

```diff
@@ -384,7 +384,35 @@ CStudioAuthoring.Dialogs.NewContentType = CStudioAuthoring.Dialogs.NewContentTyp
                 '<type>boolean</type>\r\n' +
                 '</constraint>\r\n' +
                 '</constraints>\r\n' +
-                '</field>';
+                '</field>' +
+                '<field>\r\n' +
+                '<type>input</type>\r\n' +
+                '<id>navLabel</id>\r\n' +
+                '<iceId></iceId>\r\n' +
+                '<title>Nav Label</title>\r\n' +
+                '<description></description>\r\n' +
+                '<defaultValue></defaultValue>\r\n' +
+                '<help></help>\r\n' +
+                '<properties>\r\n' +
+                '<property>\r\n' +
+                '<name>size</name>\r\n' +
+                '<value>50</value>\r\n' +
+                '<type>int</type>\r\n' +
+                '</property>\r\n' +
+                '<property>\r\n' +
+                '<name>maxlength</name>\r\n' +
+                '<value>50</value>\r\n' +
+                '<type>int</type>\r\n' +
+                '</property>\r\n' +
+                '</properties>\r\n' +
+                '<constraints>\r\n' +
+                '<constraint>\r\n' +
+                '<name>required</name>\r\n' +
+                '<value>true</value>\r\n' +
+                '<type>boolean</type>\r\n' +
+                '</constraint>\r\n' +
+                '</constraints>\r\n' +
+                '</field>\r\n';
```

Comment:
@sumerjabri could we have a place in the system to place a default content type XML definition for pages/components and use that as a dynamic default when creating a type? Eventually, we could have a default editor too.

Reply:
Yes, we can pursue that down the road. This can be a requirement for FormEngine2 (I added it to the req doc).